### PR TITLE
More AutoReplies

### PR DESCRIPTION
**Pull request by OmegaSunkey**

### CODE REVIEW

**Comment by OmegaSunkey** [comment]
Made some fixes. I'll wait for review.

File: bot/modules/auto_replies.go [hunk 1]
```diff
@@ -37,10 +37,12 @@ const (
...
+	GetSound         = "https://cdn.discordapp.com/attachments/875213883776847873/1007312170868019210/20220811_103613.mp4"
 	FullTransparency = "1. Are you using a theme that requires full transparency? If the answer is no, then that's the problem. Normally in the description says what transparency you need to use. 2. Are you using a custom ROM? If the answer is yes, then we can't do nothing about it."
 	SearchThemes     = "Check <#824357609778708580> and search on there, maybe there's the theme you want"
 	AliuCrash        = "Send crashlogs (check Crashes in Settings, and copy the most recent), if there aren't any crashlogs, then we can't do nothing about it. (If you want, you can try send a logcat, check <https://pastebin.com/pNhXwhrd>)"
-	FreeNitro        = "Not possible. Nitrospoof exists for \"free\" emotes, UserBG exists for using a custom banner (read the plugin's description), for anything else buy nitro."
+	FreeNitro        = "Not possible. Nitrospoof exists for \"free\" emotes, UserBG exists for using a custom banner (read the plugin's description), UserPFP exists for using a custom banner (read the description of that too), for anything else buy nitro."
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]: > UserPFP exists for using a custom banner

?

File: bot/modules/auto_replies.go [hunk 2]
```diff
@@ -66,21 +68,23 @@ func initAutoReplies() {
...
-		r("^(?:why|with) (?:is )?full transparency (?:is not|not|will not) (work|working)"):        FullTransparency,
+		r("^(?:i need )?help(?: me)?$"):               				    ElaborateHelp,
+		r("<@!?\\d{2,19}> help"):                       			    MentionHelp,
+		r("help <@!?\\d{2,19}>"):                                       MentionHelp,
+		r("animated (profile|avatar|pfp)"):                             FreeNitro,
+		r("is there a plugin.+"):                                       FindPlugin,
+		r("^where(?: i)?\\'?s(?: the )?.+ plugin$"):   				    FindPlugin,
+		r("^can (?:someone|anybody|anyone|you) help(?: me)?\\??$"):     JustAsk,
+		//r("can'?t (download|find|get) plugin ?downloader"): PluginDownloader, we dont need this lol
+		//r("where(?: i)s(?: the)? plugin ?downloader"):  PluginDownloader,
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]: i think we still need this
people occasionally still ask

Comment (OmegaSunkey) [reply bot/modules/auto_replies.go]: ok readding

**Comment by OmegaSunkey** [comment]
pushed changes

File: bot/modules/auto_replies.go [hunk 3]
```diff
@@ -66,21 +68,23 @@ func initAutoReplies() {
...
+		r("^(?:i need )?help(?: me)?$"):               				    ElaborateHelp,
+		r("<@!?\\d{2,19}> help"):                       			    MentionHelp,
+		r("help <@!?\\d{2,19}>"):                                       MentionHelp,
+		r("animated (profile|avatar|pfp)"):                             FreeNitro,
+		r("is there a plugin.+"):                                       FindPlugin,
+		r("^where(?: i)?'?s(?: the )?.+ plugin$"):   				    FindPlugin,
+		r("^can (?:someone|anybody|anyone|you) help(?: me)?\\??$"):     JustAsk,
+		r("(can.?not|can'?t) (download|find|get) plugin downloader"):   PluginDownloader,
+		r("where( i)?'?s( the)? plugin downloader"):                    PluginDownloader,
+		r("(?:where|how) (?:to|do i|do you) (?:install|download|get) (?:a?.?plugins?)"):                	         	  InstallPlugins,
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]:
```suggestion
		r("(?:where|how) (?:to|do i|do you) (?:install|download|get) a? ?plugins?"):                	         	  InstallPlugins,
```

File: bot/modules/auto_replies.go [hunk 4]
```diff
@@ -66,21 +68,23 @@ func initAutoReplies() {
...
+		r("how (?:to|do i|do you|i) (?:install|download|apply|get|use) (?:a?.?themes?)"):                     	    	  InstallThemes,
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]:
```suggestion
		r("how (?:to|do i|do you|i) (?:install|download|apply|get|use) a? ?themes?"):                     	    	  InstallThemes,
```

File: bot/modules/auto_replies.go [hunk 5]
```diff
@@ -66,21 +68,23 @@ func initAutoReplies() {
...
+		r("how (?:to|do i|do you|can i) (?:create|make|do) (?:a theme|(my own |a?.?custom )?themes?)"):             	  CreateThemes,
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]:
```suggestion
		r("how (?:to|do i|do you|can i) (?:create|make|do) (?:a|my own)? ?(?:custom)? ?themes?"):             	  CreateThemes,
```

File: bot/modules/auto_replies.go [hunk 6]
```diff
@@ -66,21 +68,23 @@ func initAutoReplies() {
...
+		r("how (?:to|do i|do you|can i|put) (?:change|upload|add|set) (?:sounds?|custom sounds?)"): 			 	      ThemeSounds,
+		r("how (?:to|do i|do you|can i) get sounds?(?: url| link)?"):                                                     GetSound,
+		r("(does anyone know|is there) an? (\w.+)?theme"):                                                                SearchThemes,
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]:
```suggestion
		r("(?:does anyone know|is there) an? (?:\w.+)?theme"):                                                                SearchThemes,
```

File: bot/modules/auto_replies.go [hunk 7]
```diff
@@ -66,21 +68,23 @@ func initAutoReplies() {
...
+		r("(my )?aliucord (\w.+)?(?:crashed|keeps crashing|crash|crashes|is crashing|crashing|stopped( working)?)"):      AliuCrash,
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]:
```suggestion
		r("(?:my )?aliucord (?:\w.+)?(?:crashed|keeps crashing|crash|crashes|is crashing|crashing|stopped(?: working)?)"):      AliuCrash,
```

File: bot/modules/auto_replies.go [hunk 8]
```diff
@@ -66,21 +68,23 @@ func initAutoReplies() {
...
+		r("(why |with )?(?:is )?full transparency (?:is?.?not|will not|doesn\\'t|does not) (?:work)"):                     FullTransparency,
```
Comment (rushiiMachine) [inline bot/modules/auto_replies.go]:
```suggestion
		r("(?:why |with )?(?:is )?full transparency (?:is?.?not|will not|doesn\\'t|does not) (?:work)"):                     FullTransparency,
```

**Review by rushiiMachine — CHANGES_REQUESTED** [review]

**Review by rushiiMachine — APPROVED** [review]

**Review by Juby210 — APPROVED** [review]

**Pull request merged**